media-libs/libraw/libraw-0.18.6.recipe

```diff
@@ -0,0 +1,114 @@
+SUMMARY="Library for reading and processing of RAW digicam images"
+DESCRIPTION="
+LibRaw is a library for reading RAW files from digital photo cameras \
+(CRW/CR2, NEF, RAF, DNG, MOS, KDC, DCR, etc, virtually all RAW formats are \
+supported).
+It pays special attention to correct retrieval of data required for subsequent \
+RAW conversion.
+The library is intended for embedding in RAW converters, data analyzers, and \
+other programs using RAW files as the initial data.
+"
+HOMEPAGE="http://www.libraw.org"
```

Comment:
```diff
-HOMEPAGE="http://www.libraw.org"
+HOMEPAGE="https://www.libraw.org/"
```
SOURCE_URI can also be switched to https :)

BTW, the style in LICENSE is OK, but not in COPYRIGHT and DESCRIPTION. So if you wish to follow the [HaikuPorter-Guidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)...